qiskit/visualization/circuit/text.py

```diff
@@ -1291,25 +1304,18 @@ def add_control_flow(self, node, layers, wire_map):
             # Update the wire_map with the qubits and clbits from the inner circuit
             flow_wire_map = wire_map.copy()
             flow_wire_map.update(
-                (inner, wire_map[outer]) for outer, inner in zip(node.qargs, circuit.qubits)
-            )
-            flow_wire_map.update(
-                (inner, wire_map[outer]) for outer, inner in zip(node.cargs, circuit.clbits)
+                {inner: wire_map[outer] for outer, inner in zip(node.qargs, circuit.qubits)}
             )
```

Comment:
What's the reason for dropping the clbit-binding from this?  It doesn't seem right - we still need to know how the clbits in the inner circuit map back to those in the outer circuit, right?

Reply:
No reason. Done in https://github.com/Qiskit/qiskit/pull/10842/commits/e1d7982f3177d70010e4c1e264d6c9fecd6aedc5.